en/full-node.md

```diff
@@ -0,0 +1,362 @@
+    gpg: key 8842CE5E: public key "Launchpad PPA for Bitcoin" imported
+    gpg: no ultimately trusted keys found
+    gpg: Total number processed: 1
+    gpg:               imported: 1  (RSA: 1)
+    OK
+
+Type the following line to get the most recent list of packages:
+
+    sudo apt-get update
+
+A large number of lines will be displayed as different update files are
+downloaded.  This step may take several minutes on a slow Internet
+connection.
+
+<div class="box" markdown="1">
+*Choose one of the following options*
+
+1. To install the Bitcoin Core Graphical User Interface (GUI), type the
+   following line and proceed to the [Bitcoin Core GUI](#ubuntu-gui)
+   section below:
+
+        sudo apt-get install bitcoin-qt
+
+2. To install the Bitcoin Core daemon (bitcoind), which is useful for
+   programmers and advanced users, type the following line and proceed
+   to the [Bitcoin Core Daemon](#ubuntu-daemon) section below:
+
+        sudo apt-get install bitcoind
+
+3. To install both the GUI and the daemon, type the following line and
+   read both the [GUI instructions](#ubuntu-gui) and the [daemon
+   instructions](#ubuntu-daemon). Note that you can't run both the GUI
+   and the daemon at the same time using the same configuration
+   directory.
+
+        sudo apt-get install bitcoin-qt bitcoind
+
+After choosing what packages to install, you will be asked whether you
+want to proceed.  Press enter to continue.
+</div>
+
+#### Bicoin Core GUI {#ubuntu-gui}
+
+To start Bitcoin Core GUI, click the Ubuntu swirl icon to open the Dash,
+type `bitcoin`, and click the Bitcoin icon.
+
+![Dash Bitcoin-Qt](/img/full-node/en-dash-bitcoin-qt.png)
+
+You will be prompted to choose a directory to store the Bitcoin block
+chain and your wallet.  Unless you have a separate partition or drive
+you want to use, click Ok to use the default.
+
+![Bitcoin-Qt Welcome](/img/full-node/en-bitcoin-qt-welcome.png)
+
+Bitcoin Core GUI will begin to download the block chain.  This
+step will take at least several hours, and it may take a day or more on
+a slow Internet connection or with a slow computer.  During the
+download, Bitcoin Core will use a significant part of your connection
+bandwidth.  You can stop Bitcoin Core at any time by closing it; it will
+resume from the point where it stopped the next time you start it.
+
+![Bitcoin-Qt Initial Block Download](/img/full-node/en-bitcoin-qt-ibd.png)
+
+After download is complete, you may use Bitcoin Core as your wallet or
+you can just let it run to help support the Bitcoin network.
+
+<div class="box" markdown="1">
+*Optional: Start Your Node At Login*
+
+Starting your node automatically each time you login to your computer
+makes it easy for you to contribute to the network. The easiest way
+to do this is to add Bitcoin Core GUI to the list of startup
+applications.
+
+Click the Ubuntu swirl icon to start the Dash, type `startup app`,
+and click the Startup Applications icon.
```

Comment:
Bitcoin Core has a "Start Bitcoin Core on startup" option in it's own preferences box on Linux. This could maybe cut a few steps and be more compatible with other Linux distributions?